locale/core/german.json

```diff
@@ -0,0 +1,37 @@
+{
+	"AND": "und",
+
+	"SECONDS": "Sekunden",
+	"SECOND": "Sekunde",
+	"SECOND_SHORT": "s",
+	"MINUTES": "Minuten",
+	"MINUTE": "Minute",
+	"MINUTE_SHORT": "m",
+	"HOURS": "Stunden",
+	"HOUR": "Stunde",
+	"HOUR_SHORT": "h",
```

Comment:
Should this be 'S'? This is a one-character way to refer to hours, so in English, "12h" would mean 12 hours. I realize that 's' is taken by 'seconds', so does 'S' make more sense here? In context, in German, "12S 15m 19s" would mean 12 hours, 15 minutes, 19 seconds.